### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by hugocoding21**

### CODE REVIEW

File: backend/src/models/UsersManager.js [hunk 1]
```diff
@@ -18,6 +18,20 @@ class UsersManager extends AbstractManager {
...
   }
-}
 
+  login(user) {
+    return this.database.query(`SELECT * FROM ${this.table} WHERE mail = ?`, [
+      user.mail,
+    ]);
+  }
+}
+//   login(user) {
```
Comment (fantinerudent) [inline backend/src/models/UsersManager.js:28]: no dead code

File: frontend/src/page/FormLogin/FormLogin.jsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    axios
+      .post("http://localhost:5000/login", formData, {
+        withCredentials: true,
+      })
+      .then(() => {
+        // console.log("Requête réussie :", response);
```
Comment (fantinerudent) [inline frontend/src/page/FormLogin/FormLogin.jsx:24]: no dead code

**Review by fantinerudent — CHANGES_REQUESTED** [review]

**Review by mcgrathmichael — DISMISSED** [review]

File: backend/src/router.js [hunk 3]
```diff
@@ -1,45 +1,59 @@
 const express = require("express");
 
 const router = express.Router();
+const {
+  hashPassword,
+  verifyPassword,
+  // verifyToken,
```
Comment (fantinerudent) [inline backend/src/router.js:7]: dead code

File: frontend/src/components/LoginForm/LoginRegisterForm.jsx [hunk 4]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (fantinerudent) [inline frontend/src/components/LoginForm/LoginRegisterForm.jsx:1]: why

Comment (fantinerudent) [inline frontend/src/components/LoginForm/LoginRegisterForm.jsx:1]: nop

**Review by fantinerudent — APPROVED** [review]

**Pull request merged**